### ISSUE TITLE
I might be doing something wrong, but when I put the code into a bookmarklet, it says "invalid URL". How do I fix this?

### ISSUE THREAD

**Issue by Gion2** [ecc521/gimkit-bot#40]

**Comment by ecc521** [comment]
I no longer maintain this project. Can @TreeBronch help at all?

**Comment by Gion2** [comment]
Well by the time this was answered, I got it working. But the bookmarklet doesn't work. When I put in "s", nothing happens.

**Comment by Esb678** [comment]
Do “JavaScript:” before you type the code

**Comment by MikeIsCoolio** [comment]
Is it even able to work on a mobile device?

**Comment by Justiniscoding** [comment]
> Is it even able to work on a mobile device?

Yes, bookmarklet works on iPads and I presume they do on iPhones too
